pkg/cmd/bootstrap_test.go

```diff
@@ -290,13 +284,13 @@ func TestDependenciesWithNoPipelines(t *testing.T) {
 	wantMsg := `
 Checking if Sealed Secrets is installed with the default configuration
 Checking if ArgoCD Operator is installed with the default configuration
-Checking if OpenShift Pipelines Operator is installed with the default configuration[Please install OpenShift Pipelines operator from OperatorHub]`
+Checking if OpenShift Pipelines Operator is installed with the default configuration [Please install OpenShift Pipelines operator from OperatorHub]`
 
 	buff := &bytes.Buffer{}
 	fakeSpinner := &mockSpinner{writer: buff}
 	wizardParams := &BootstrapParameters{&pipelines.BootstrapOptions{}}
-	err := checkBootstrapDependencies(wizardParams, fakeClient, fakeSpinner)
-	wantErr := "Failed to satisfy the required dependencies"
+	_, err := checkBootstrapDependencies(wizardParams, fakeClient, fakeSpinner)
+	wantErr := "failed to satisfy the required dependencies: OpenShift Pipelines Operator"
```

Comment:
Not sure why it starts with lowercase.

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#error-strings